spec/units/peeps_spec.rb

```diff
@@ -10,11 +10,22 @@
       add_peeps
     end
     it 'adds a peep to the peeps database' do
-      Peeps.post_peep('A peep','1')
+      Peeps.post_peep('A peep', '1')
       expect(Peeps.all[0].text).to eq 'A peep'
     end
     it 'sorts peeps by date' do
       expect(Date.parse(Peeps.all[0].date)).to be > Date.parse(Peeps.all[1].date)
     end
+    it 'retrieves all peeps from the database' do
+      expect(Peeps.all.length).to eq 3
+    end
+    it 'skips a Peep if there is no text' do
+      Peeps.post_peep('', '1')
+      expect(Peeps.all.length).to eq 3
+    end
+    it 'removes apostrophes from a tweet' do
```

Comment:
great idea to get rid of these! did not get around to doing this myself, but really helps to prevent bugs